package.json

```diff
@@ -110,6 +110,7 @@
     "@types/node": "^17.0.22",
     "@typescript-eslint/eslint-plugin": "^5.16.0",
     "@typescript-eslint/parser": "^5.16.0",
+    "autoprefixer": "^10.4.19",
```

Comment:
I'm confused by this - it wasn't part of any package.json file, yet it's called in the Gulp files. Most other packages referenced there were included in this list, but not this one.

After bumping Stylelint and changing the config, `yarn build:core` starting screaming at me about not being able to find this module. Maybe it was part of another package I and it somehow just worked? Either way, adding it here resolved the build issues.

Reply:
Nice find 👍